### PR TITLE
[NFS] nfs coredump

### PR DESCRIPTION
**Pull request by MohitBis**
[NFS] nfs coredump

### CODE REVIEW

File: tests/nfs/nfs_verify_file_operations.py [hunk 1]
```diff
@@ -118,3 +119,28 @@ def run(ceph_cluster, **kw):
         cleanup_cluster(clients, nfs_mount, nfs_name, nfs_export)
         log.info("Cleaning up successfull")
     return 0
+
+
+def nfs_coredump(
```
Comment (pranavprakash20) [inline tests/nfs/nfs_verify_file_operations.py:124]: can we rename to check_nfs_coredump ?

Comment (MohitBis) [reply tests/nfs/nfs_verify_file_operations.py]: Updated

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**